saleor/page/models.py

```diff
@@ -22,6 +23,7 @@ class Page(ModelWithMetadata, SeoModel, PublishableModel):
     class Meta(ModelWithMetadata.Meta):
         ordering = ("slug",)
         permissions = ((PagePermissions.MANAGE_PAGES.codename, "Manage pages."),)
+        indexes = [GinIndex(fields=["title", "slug"])]
```

Comment:
We should remove `ModelWithMetadata.Meta`  from `Meta` inheritance and :
```suggestion
        indexes = [
            *ModelWithMetadata.Meta.indexes,
            GinIndex(fields=["title", "slug"])
        ]
```